testsuite/features/step_definitions/api_common.rb

```diff
@@ -238,7 +238,7 @@
 When(/^I create an activation key including custom channels for "([^"]*)" via API$/) do |client|
   # Create a key with the base channel for this client
   id = description = "#{client}_key"
-  base_channel = LABEL_BY_BASE_CHANNEL[BASE_CHANNEL_BY_CLIENT[client]]
+  base_channel = get_os_channel_from_map(LABEL_BY_BASE_CHANNEL, get_os_channel_from_map(BASE_CHANNEL_BY_CLIENT, client))
```

Comment:
Not a fan of using getters when an arry just works

Reply:
Yeah... maybe in this case we don't have any benefit.
I would agree, if we keep the current data structure, to just use 👍🏼 
```
 base_channel_map = BASE_CHANNEL_BY_CLIENT[product][client]
 base_channel = LABEL_BY_BASE_CHANNEL[product][base_channel_map]
```
And maybe.. we might need to rename the name of the BASE_CHANNEL_BY_CLIENT constant.

Reply:
Fixed.